common/lib/dependabot/pull_request_creator/labeler.rb

```diff
@@ -193,7 +193,8 @@ def security_label_exists?
       end
 
       def security_label
-        labels.find { |l| l.match?(/security/i) }
+        labels.find { |l| l == DEFAULT_SECURITY_LABEL } ||
+          labels.find { |l| l.match?(/security/i) }
       end
```

Comment:
Could you also add a test case for this in the labeler_spec? You could do something similar to this: https://github.com/dependabot/dependabot-core/commit/503c77e1bcda22d9c06185afca0a8c61373307c4#diff-27da36e17c0a714c6d5db6824b28b4fe but adding `let(:includes_security_fixes) { true }` to the context.